hangups/default.conf

```diff
@@ -0,0 +1,11 @@
+[General]
+col-scheme: default
+
+[Keybindings]
+key-prev-tab: ctrl u
+key-next-tab: ctrl d
+key-close-tab: ctrl w
+key-quit: ctrl e
+key-menu: ctrl n
+key-up: k
+key-down: j
```

Comment:
Oh. I thought the root working directory of the application, after installation, would be the hangups directory (under the root of the repo directory). At least that's what it looks like on my system, e.g., the `ui` directory is at `/usr/lib/python3.5/site-packages/hangups/ui/`. If we put default.conf in the root of the repository, where would it end up after installation?